make_queue/util/user_info.py

```diff
@@ -0,0 +1,53 @@
+from django.contrib.auth.models import User
+import ldap
+
+LDAP_HOST = 'ldap://at.ntnu.no'
```

Comment:
Could we move this file to the `dataporten` app with a file name of `ldap.py`? It feels more natural to collect all the external user information sources in one app. Additionally, adding [docstrings](https://www.python.org/dev/peps/pep-0257/) would be great.

Reply:
Moved to `dataporten/ldap_utils.py` to not shadow the `python-ldap` package.